yt/data_objects/particle_trajectories.py

```diff
@@ -111,15 +111,15 @@ def __init__(
             mylog.setLevel(old_level)
 
         times = []
-        for fn, (time, indices, pfields) in sorted(my_storage.items()):
+        for _fn, (time, _indices, _pfields) in sorted(my_storage.items()):
             times.append(time)
         self.times = self.data_series[0].arr([time for time in times], times[0].units)
```

Comment:
wow, I didn't know about the syntax you use in the second version. Is there a name for this ? I'm tempted to use it because it's new and shiny but I'm not sure about readability.

Reply:
For future reference, this is named "Extended Iterable Unpacking", see https://www.python.org/dev/peps/pep-3132/.